test/color/setting_test.dart

```diff
@@ -180,6 +180,68 @@ void main() {
 
         await screenMatchesGolden(tester, 'color_setting_nostroke-example-1');
       });
+
+      processingLegacySpecTest('clear() replaces background in setup() and drawn rect', (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
+              setup: (s) {
+                s.background(color: const Color(0xFF404040));
+              },
+              draw: (s) {
+                s
+                  ..noLoop()
+                  ..fill(color: const Color(0xFF969696))
```

Comment:
This `fill() + rect()` line still prevents the thing that you're trying to test. The `background()` in `draw()` covers the earlier `background()` in the `setup()` call. If you want to ensure that `clear()` replaces the setup background, then you should remove the `fill()` and `background()` calls in `draw()`.

Reply:
@ThangVuNguyenViet it looks like this still needs to be addressed, right?